frontend/src/metabase/admin/routes.jsx

```diff
@@ -170,6 +178,20 @@ const getRoutes = (store, CanAccessSettings, IsAdmin) => (
         {getAdminPermissionsRoutes(store)}
       </Route>
 
+      <Route path="tools" component={IsAdmin}>
```

Comment:
Please replace `IsAdmin` with createAdminRouteGuard("tools") to allow non-admin users with the monitoring permission open the route

Reply:
Oh, got it, thanks for investigating that!